rust_src/src/buffers.rs

```diff
@@ -163,6 +163,10 @@ impl LispBufferRef {
         self.filename_
     }
 
```

Comment:
These accessors are a relic from before we had bindgen. Does having this method help in a significant way?

Reply:
No, I just copied the existing style. Those `field_` names are pretty ugly though, so I think this looks better. Have we settled on a standard way of doing this?

Reply:
Okay, I got rid of it.